### PR TITLE
[DEVX-1634] Add saucectl documentation for excludedTestFiles

### PR DESCRIPTION
**Pull request by tianfeng92**
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [x] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by alexplischke — APPROVED** [review]

**Review by kimsauce — CHANGES_REQUESTED** [review]
Should the wording for these be more consistent?

File: docs/web-apps/automated-testing/testcafe/yaml.md [hunk 1]
```diff
@@ -688,6 +688,16 @@ The explicit name, file glob, or location of the test files to be included in th
 ```
 ---
 
+### `excludedTestFiles`
+<p><small>| OPTIONAL | ARRAY/REGEX |</small></p>
+
+Excludes test files to skip the tests. Files are matched by shell pattern, e.g. the explicit name, file glob, or the location of the test files.
```
Comment (kimsauce) [inline docs/web-apps/automated-testing/testcafe/yaml.md:694]:
```suggestion
Excludes test files to skip the tests. Files are matched by shell pattern, such as the explicit name, file glob, or location of the test files.
```

File: docs/web-apps/automated-testing/playwright/yaml.md [hunk 2]
```diff
@@ -644,6 +644,16 @@ One or more paths to the playwright test files to run for this suite. Regex valu
 ```
 ---
 
+### `excludedTestFiles`
+<p><small>| OPTIONAL | ARRAY |</small></p>
+
+Excludes test files to skip the tests. Regex values are supported to indicate all files of a certain type or in a certain directory, etc.
```
Comment (kimsauce) [inline docs/web-apps/automated-testing/playwright/yaml.md:650]: @lysannep @tianfeng92 Can you take a look at this wording? 

```suggestion
Excludes test files to skip the tests. You can use regex values to indicate all files that match a specific value, such as a file name, type, or directory.
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
A couple comments!

**Review by kimsauce — APPROVED** [review]

**Pull request merged**